<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Mallis-Mac-mini.local" name="SmokeRegistration" tests="5" failures="0" timestamp="7 Jul 2017 19:16:35 GMT" time="1.556" errors="0">
  <testcase name="@BeforeClass setUp" time="0.0" classname="testclasses.IosAdminBoardsPageTest">
    <failure type="org.openqa.selenium.WebDriverException" message="org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:4723 [/127.0.0.1] failed: Connection refused (Connection refused)
Build info: version: &amp;apos;3.4.0&amp;apos;, revision: &amp;apos;unknown&amp;apos;, time: &amp;apos;unknown&amp;apos;
System info: host: &amp;apos;Mallis-Mac-mini.local&amp;apos;, ip: &amp;apos;fe80:0:0:0:1c62:425f:5620:c0f4%en0&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.12.4&amp;apos;, java.version: &amp;apos;1.8.0_131&amp;apos;
Driver info: driver.version: IOSDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:4723 [/127.0.0.1] failed: Connection refused (Connection refused)
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'Mallis-Mac-mini.local', ip: 'fe80:0:0:0:1c62:425f:5620:c0f4%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.12.4', java.version: '1.8.0_131'
Driver info: driver.version: IOSDriver
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:84)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:637)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:51)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:236)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:137)
	at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:47)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:114)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:132)
	at io.appium.java_client.ios.IOSDriver.<init>(IOSDriver.java:81)
	at utilities.BaseClass.iOSMobileNativeAppCapabilities(BaseClass.java:36)
	at testclasses.IosAdminBoardsPageTest.setUp(IosAdminBoardsPageTest.java:42)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:4723 [/127.0.0.1] failed: Connection refused (Connection refused)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:139)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:87)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:343)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:159)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:69)
	... 37 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	... 52 more
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@AfterClass tearDown" time="0.005" classname="testclasses.IosAdminBoardsPageTest">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="verifyMessage" time="0.001" classname="testclasses.IosAdminBoardsPageTest">
    <skipped/>
  </testcase> <!-- verifyMessage -->
  <testcase name="verifyWriteCommentEithValidData" time="0.001" classname="testclasses.IosAdminBoardsPageTest">
    <skipped/>
  </testcase> <!-- verifyWriteCommentEithValidData -->
  <testcase name="verifyWriteCommentEithInValidData" time="0.001" classname="testclasses.IosAdminBoardsPageTest">
    <skipped/>
  </testcase> <!-- verifyWriteCommentEithInValidData -->
  <testcase name="verifyConvertTaskWithValidData" time="0.002" classname="testclasses.IosAdminBoardsPageTest">
    <skipped/>
  </testcase> <!-- verifyConvertTaskWithValidData -->
  <testcase name="verifyConvertTaskWithBlankData" time="0.005" classname="testclasses.IosAdminBoardsPageTest">
    <skipped/>
  </testcase> <!-- verifyConvertTaskWithBlankData -->
</testsuite> <!-- SmokeRegistration -->
